### DISCUSSION TITLE
Test version

### DISCUSSION THREAD

**Discussion by Tabu82** [brutella/hkknx-public#214] [Q&A]
Hi everybody,
maybe someone can help me out. I installed the bridge on an pi according to the documentation.
Furthermore, I uploaded a test project which probably has more than 15 devices.
Unfortunately,  I can not see any devices only the approx. 200 group addresses. 
Furthermore, I am not able to update the version via the html page. The version is stated as [1.3.0](2021-05-20T12:24:19+02:00) 

Do you have any idea? 
Best regards and thanks in advance,
Tabu

[a comment by an automated account is not shown]

**Comment by Tabu82** [comment]
Hi,

Followed the instruction with automatic installation via the image.  Thanks for the help!

[a comment by an automated account is not shown]

**Comment by Tabu82** [comment]
Hello,
I tried to update the version according to the homepage. MY current version is stated as 1.3 and if I hit for search for new updates  I get the following error message: Get "https://api.github.com/repos/brutella/hkknx-public/releases": x509: certificate has expired or is not yet valid: current time 2021-05-20T15:26:57+01:00 is before 2022-03-16T00:00:00Z
Thanks for the help

**Reply by chef76** [reply]
If you use the PI Image you have to set your System Time.

**Reply by Tabu82** [reply]
Hi, 

thank you for the response.

I modified the script for timing according to my understanding
![Bild 19 09 22 um 12 26](https://user-images.githubusercontent.com/113640903/190998236-f88b0e18-24c0-4484-b071-e963370dffde.jpg)

I tried using the command to start the service but didn't work out
_timedatectl set-ntp 1_

Can you help me?

**Reply by Tabu82** [reply]
Thanks for the  help. The manual overwrite did work for me and I was able to update to most recent version.
Nevertheless, I got two questions

1. The automatic service for time sync. doesn't seem to work for me:
![Bild 19 09 22 um 12 53](https://user-images.githubusercontent.com/113640903/191004515-7813bbfe-03eb-46f8-8c22-b48b0800d9b1.jpg)

2. I was able to import my KNX-project but see no devices only the group addresses. You have any ideas:
![Bild 19 09 22 um 13 06](https://user-images.githubusercontent.com/113640903/191005065-506422da-02ee-4366-abd0-f3ad4eb59e28.jpg)

**Reply by Tabu82** [reply]
Many thanks for the response. 
My lamp now would look as follows: 
![image](https://user-images.githubusercontent.com/113640903/191016875-18f2a3ac-6ae1-48f1-abb4-e4928afe8400.png)
Do you see any problems with that description?

Furthermore , i tried to connect to my IP-Interface which can not be found. In the description it is mentioned  that multicast need to be available.  How can i active this? 

Thank you in advance

[a comment by an automated account is not shown]

**Reply by Tabu82** [reply]
Thank you again. 
My KNX-Interface is IPS300SREG from Albrecht Jung. 
The ip-address is fixed to 192.168.178.88 
When trying to connect it states unsuccessful

![image](https://user-images.githubusercontent.com/113640903/191025538-3958c7d4-a80c-4ba8-b357-cbb6c46638e8.png)

**Reply by Tabu82** [reply]
This i at least now could figure out…my TCP-IP Interface is now broken…need to order new one…maybe afterwards it will work

**Reply by Tabu82** [reply]
The Display on the device is offline, the Interface was not responding when pushing the buttons by hand neither was it online in FRITZ!Box. 

But yeah i agree - nothing to do with the bridge- just bad timing need to order new device than hopefully i can make it work.

[a comment by an automated account is not shown]
